rotkehlchen/exchanges/coinbase.py

```diff
@@ -344,22 +447,15 @@ def _api_query(
             if 'data' not in json_ret:
                 raise RemoteError(f'Coinbase json response does not contain data: {response.text}')
 
-            # `data` attr is a list in itself
-            all_items.extend(json_ret['data'])
+            all_items.extend(json_ret['data'])  # `data` attr is a list in itself
             if ignore_pagination or 'pagination' not in json_ret:
-                # break out of the loop, no need to handle pagination
-                break
+                break  # break out of the loop, no need to handle pagination
 
             if 'next_uri' not in json_ret['pagination']:
                 raise RemoteError('Coinbase json response contained no "next_uri" key')
 
-            # otherwise, let the loop run to gather subsequent queries
```

Comment:
Again as I said before. Why are you removing this comment here? Please do not touch it.

Reply:
Added back comments! Thank you.